ocaml/xapi/helpers.ml

```diff
@@ -1230,3 +1230,32 @@ let host_supports_hvm ~__context host =
 let env_with_path env_vars =
   Array.of_list @@
   List.map (fun (k,v) -> Printf.sprintf "%s=%s" k v) (("PATH", String.concat ":" Forkhelpers.default_path)::env_vars)
+
+let are_on_same_subnet_exn netmask ips =
```

Comment:
Would a `bool` as a result not be as good? The name indicates that this a predicate. On the client side we would have 
```
match Helpers.are_on_same_subnet_exn netmask ips with
| true ->
| false ->
```

(or using an `if`) - I don't think there could be confusion of the meaning with the name chosen as is.